lib/puma/client.rb

```diff
@@ -613,5 +613,11 @@ def set_ready
       @requests_served += 1
       @ready = true
     end
+
+    def above_http_content_limit(value)
+      return false if @http_content_length_limit.nil?
+
+      value > @http_content_length_limit
+    end
```

Comment:
Maybe the following?   Looks a bit odd, but...
```ruby
  def above_http_content_limit(value)
    @http_content_length_limit&.< value
  end
```

Reply:
sure, updated! 🙌

Reply:
Thanks.  I'm busy with some cluster-f*(& code due to XMas Ruby releases.  Thanks for adding code for chunked.  I can't review till later, maybe someone else can...

Reply:
haha! no rush, thanks for the suggestions and supporting xmas ruby releases. Have a fun xmas break! 🎄